src/halmos/__main__.py

```diff
@@ -1449,10 +1447,6 @@ def _main(_args=None) -> MainResult:
     timer = NamedTimer("total")
     timer.create_subtimer("build")
 
-    # clear any remaining live display before starting a new instance
-    rich.get_console().clear_live()
-    progress_status.start()
```

Comment:
moving the status start to after the forge build command, which itself has a live status and they seemed to clash